components/navbar.js

```diff
@@ -2,10 +2,10 @@ import { Nav, Navbar, NavDropdown, NavItem } from "react-bootstrap";
+                <svg height="42" viewBox="0 0 469 117" fill="#F1F1F1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We should move this svg out to it's own file so that the navbar file isn't so cluttered